htmlbook-xsl/xspec/common.xspec

```diff
@@ -1251,11 +1251,83 @@ sect5:none
 	    <h1>Subsection heading</h1>
 	    <p>Always soft-code your cross-references</p>
 	    <p>Like this one to the sidebar in the previous chapter: see <a class="underline" id="sidebar_xref" data-type="xref" href="#sidebar"/></p>
+	  	<p>This is an xref to a refentry: see <a id="refentry_no_title_xref" data-type="xref" href="#refentry"/></p>

```

Comment:
The `href` for the XREF on line 1254 should be `href="#refentry_no_title"`. (Note: this doesn't have any effect on the tests, as they use the `xref.target` param to determine the target, not the `href`, but still worth fixing for correctness and to avoid confusion)